### PR TITLE
docs: firmware flashing instruction + mention V3 is ported

### PR DESCRIPTION
**Pull request by thiswillbeyourgithub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,10 +166,15 @@ A recent update  reduces the voltage supply from 3.3V to 2.7V  during sleep, the
...
-
-
-
-
-
-
- 
+### How to firmware flash
+
+* Install the latest **[thonny](https://thonny.org/)**
```
Comment (devnoname120) [inline README.md:182]: You may want to consider adding instructions for using `esptool` alone as an alternative to Thonny for flashing the firmware: https://docs.micropython.org/en/v1.20.0/esp32/tutorial/intro.html#deploying-the-firmware

File: README.md [hunk 2]
```diff
@@ -167,10 +166,15 @@ A recent update  reduces the voltage supply from 3.3V to 2.7V  during sleep, the
```
Comment (devnoname120) [inline README.md:167]: @jeffmer How can we measure the light sleep current of V3?

**Review by devnoname120 — APPROVED** [review]
Thanks for your updates!

File: README.md (hunk 1 above)
Comment (thiswillbeyourgithub) [reply README.md:182]: Oh neat! I will add that on monday.

File: README.md (hunk 2 above)
Comment (jeffmer) [reply README.md:167]: To measure it, I took out the battery and connected some thin wires to the battery connectors on the watch and then used an external lipo battery in series with a multimeter to measure the current.

File: README.md (hunk 1 above)
Comment (thiswillbeyourgithub) [reply README.md:182]: I added the example.

File: README.md (hunk 2 above)
Comment (thiswillbeyourgithub) [reply README.md:167]: Is the mAmp value appearing in the "battery" app reliable? My V3 displays something between 20 and 45 mA btw

Comment (jeffmer) [reply README.md:167]: I don’t think it is very accurate but it does increase if you turn up brightness or CPU frequency so it’s a useful indicator.

Comment (thiswillbeyourgithub) [reply README.md:167]: Indeed. Btw my watch battery died during the night. It was at 4.13V at 7pm and dead when I checked at 2pm today.

ps : sorry if I added the commits from the other branch to this branch, I'm not always super comfortable with git and github so yeah

Comment (thiswillbeyourgithub) [reply README.md:167]: @devnoname120 helped me fix this PR commit history. AFAIC this is ready to be merged.

**Comment by jeffmer** [comment]
Many thanks for your work in testing and documenting V3

**Comment by thiswillbeyourgithub** [comment]
Lots more to come :)

**Pull request merged**